### DISCUSSION TITLE
电脑开机后经常出现这种情况Post "https://openapi.aliyundrive.com/adrive/v1.0/user/getDrivelnfo": dial tcp: lookup openapi.aliyundrive.com: no such host

### DISCUSSION THREAD

**Discussion by YuanKun666** [AlistGo/alist#4592] [General]
<img width="296" alt="image" src="https://github.com/alist-org/alist/assets/76767255/548d2b48-00cf-42d1-9460-3600ad00b63f">

**Comment by xuezhonghai** [comment]
将配置里的"Oauth token url"项中的 https://api.nn.ci 改成 https://api-cf.nn.ci 或者 https://api.xhofe.top 试试。
